src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -98,12 +98,14 @@ def delete_fs_volume(self, volname, confirm):
                 "that is what you want, re-issue the command followed by " \
                 "--yes-i-really-mean-it.".format(volname)
 
-        ret, out, err = self.mgr.check_mon_command({
+        ret, out, err = self.mgr.mon_command({
```

Comment:
from the test failure looks like this is incorrect usage of the method.

Reply:
I tested it locally. The config command is not taking 'mon', it is expecting specific mon, like mon.0. 
It works with 'mon.*' though, so please change it and it should work.